pkg/limayaml/limayaml_test.go

```diff
@@ -0,0 +1,43 @@
+package limayaml
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func dumpJSON(d interface{}) string {
+	b, err := json.Marshal(d)
+	if err != nil {
+		return "ERROR"
+	}
+	return string(b)
+}
```

Comment:
We can abort a test on marshal failure:
```suggestion
func dumpJSON(t *testing.T, d interface{}) string {
	b, err := json.Marshal(d)
	if err != nil {
		t.Fatal(err)
	}
	return string(b)
}
```